cluster/distributedtask/scheduler.go

```diff
@@ -0,0 +1,365 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package distributedtask
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/pkg/errors"
```

Comment:
same here, can we use `fmt.Errorf(...)` instead of this old and archived package?

Reply:
yes, refactored